src/main.cpp

```diff
@@ -193,35 +253,49 @@ void wifi_sniffer_loop(void * pvParameters) {
 
   	while (true) {
         nloop++;
-		vTaskDelay(cfg.wifichancycle*10 / portTICK_PERIOD_MS);
+        vTaskDelay(cfg.wifichancycle*10 / portTICK_PERIOD_MS);
         yield();
-		wifi_sniffer_set_channel(channel);
-		channel = (channel % WIFI_CHANNEL_MAX) + 1;
+        wifi_sniffer_set_channel(channel);
+        channel = (channel % WIFI_CHANNEL_MAX) + 1;
+        ESP_LOGI(TAG, "Wifi set channel %d", channel);
+        
+        u8x8.setCursor(0,5);
+        u8x8.printf(!cfg.rssilimit ? "RLIM:  off" : "RLIM: %4i", cfg.rssilimit);
+        u8x8.setCursor(11,5);
+        u8x8.printf("ch:%02i", channel);

```

Comment:
Nice idea!
But i'm not sure if this will work stable. I had continuous problems with occasionally garbage on display. Not sure what the root cause is (maybe timing on I2C or race condition due to RTos task?), but it seems this problem rises as writes to display increase.
-> To be checked.